src/app/shared/models/dot-tag/dot-tag.model.ts

```diff
@@ -0,0 +1,6 @@
+export interface DotTag {
+    label: string;
+    siteId?: string;
```

Comment:
Why all this are not required? we have to stop this practice.

Reply:
fixed